Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,346 @@
-/* Style applied to the BocList. */
-
 div.bocList
 {
-  display: block;
-  position: relative;
-  zoom: 1;
-  height: auto;
-  width: auto; /* give it a value to make bocList scrollable, IE7 is buggy if you don't define height & width */
-}
-
-/* Style applied to the BocList in read-only mode. */
-.bocList.readOnly
-{
-}
-
-/*  Style applied to the BocList's navigator. */
-div.bocListNavigator
-{
-  white-space: nowrap;
-  overflow: hidden;
-  margin-top: 0.25em;
-}
-
-
-div.bocList.hasDimensions div.bocListNavigator
-{
-  position: absolute;
-  bottom: 0;
+  --boclist-tablehead-background-color: var(--color-contrast);
+  --boclist-listmenu-background-color: var(--color-contrast);
+  --boclist-tablecell-border-color: var(--color-standard-separator);
+  --boclist-menublock-minimum-width: 10em;
+  --boclist-menublock-maximum-width: auto;
 }
 
-/*  Style applied to the BocList's navigator, page information. */
-div.bocListNavigator > span
-{
-  padding-right: 0.35em
-}
-
-div.bocListNavigator > span > input
-{
-  margin-left: 0.35em;
-  margin-right: 0.35em;
-}
-
-div.bocListNavigator a
-{
-  padding: 2px;
-}
-
-div.bocListNavigator a[href]
-{
-  color: #09457E;
-}
-
-div.bocListNavigator a[href]:focus,
-div.bocListNavigator a[href]:hover
-{
-  color: Black;
-  padding: 1px;
-  border: 1px solid #7CB2E6;
-  background-color: #E5F0FC;
-  outline: none;
-}
-
-div.bocListNavigator a[href]:active
-{
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
-}
-
-div.bocListNavigator img
-{
-  padding-left: 0.35em;
-  padding-right: 0.35em;
-  vertical-align: -0.1em;
-}
-
-div.bocListNavigator a[id$="_First"] img,
-div.bocListNavigator a[id$="_Last"] img
-{
-  height: 16px;
-  width: 16px;
-}
-
-div.bocListNavigator a[id$="_Previous"] img,
-div.bocListNavigator a[id$="_Next"] img
-{
-  height: 16px;
-  width: 16px;
-}
-
-div.bocListNavigator a[href]:focus img,
-div.bocListNavigator a[href]:hover img,
-div.bocListNavigator a[href]:active img
-{
-  -webkit-filter: brightness(0);
-  filter: brightness(0);
-  filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
-}
-
-/*  Style applied to the BocList's menu block. */
-div.bocListMenuBlock
+/*region root*/
+div.bocList
 {
-  display: block;
-  float: right;
-  width: 10em;
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--boclist-menublock-minimum-width), var(--boclist-menublock-maximum-width));
+  height: auto;
 }
+/*endregion*/
 
-div.bocListMenuBlock > div
+/*region bocListTableContainer*/
+div.bocList div.bocListTableBlock.hasNavigator div.bocListTableContainer
 {
-  margin-left: 0.5em;
+  grid-row: 1;
 }
 
-/*  Style applied to the list-area (table and navigation block) of the BocList.  */
-div.bocListTableBlock
+div.bocList div.bocListTableBlock div.bocListTableContainer
 {
-  display: block;
-  width: auto;
+  grid-row: 1 / span 2;
+  position: relative;
+  background-color: var(--boclist-tablehead-background-color);
   overflow: hidden;
 }
+/*endregion*/
```

Comment:
```suggestion
  overflow: hidden;
}

div.bocList.hasDimensions  div.bocListTableBlock div.bocListTableContainer
{
  background-color: var(--boclist-tablehead-background-color);
}

/*endregion*/
```

Sorry, just found a place where this is a problem, but can solve this by only applying the style when the BocList has dimensions (i.e. a scrollcontainer)
![image](https://user-images.githubusercontent.com/6972548/133459336-e1830b50-ce52-480e-8f53-398524387e0d.png)
(red for better visibility)